lib/Serializer.js

```diff
@@ -0,0 +1,118 @@
+class Serializer {
```

Comment:
How do we feel about creating a folder called `Serializer`. Then having this file be called `Serializer/index.js`. Then having a file called `Serializer/utils.js`. That way we can separate it out a bit more.

Reply:
Im not sure that's very necessary as its a pretty small class in itself. The injection of the methods on the other object is a bit weird perhaps but being only 3 lines or less is pretty manageable to hook into everything, and really its just for ease of access to the user instead of going to the Model.serializer object itself every time.